### PR TITLE
Really make cron job hourly

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt r? (again, sorry!)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ pipeline {
   }
   triggers {
     pollSCM('H/5 * * * *')
-    cron('@daily')
+    cron('H H * * *')
```
Comment (davehunt) [inline Jenkinsfile]: hah, so this would be daily, which is what I had assumed you wanted.. I believe hourly would be H * * *

Comment (stephendonner) [reply Jenkinsfile]: Sigh; I need to sleep (really).  Fixing.

The good news is that I can verify the two values were both set in the UI, automagically!

screen shot 2018-02-13 at 7 34 40 am

**Comment by stephendonner** [comment]
@davehunt sorry, again - should be good to go

**Comment by rpappalax** [comment]
👍

**Pull request merged**